src/org/spdx/licensexml/LicenseXmlHelper.java

```diff
@@ -311,11 +317,20 @@ private static void appendOptionalText(Element element,
 			sb.append(childSb);
 			sb.append("<<endOptional>>");
 		} else if (includeHtmlTags) {
-			sb.append("\n<span class=\"");
+			if (includesPhrasing(element)) {
+				sb.append("\n<div class=\"");
+			} else {
+				sb.append("\n<var class=\"");
+			}
+			
 			sb.append(HtmlTemplateOutputHandler.OPTIONAL_LICENSE_TEXT_CLASS);
 			sb.append("\">");
 			sb.append(childSb.toString());
-			sb.append("</span>");
+			if (includesPhrasing(element)) {
+				sb.append("</div>");
```

Comment:
This could be a bit DRYer and slightly more performant if you had a single:

```java
if (includesFlowContent(element)) {
  tag = "div";
} else {
  tag = "var";
}
```

and then used that `tag` variable for both the start and end tag.

Reply:
Agree - I was just being lazy ;)  I'll update the PR